mcs/class/System.Data/corefx/SR.cs

```diff
@@ -786,7 +786,7 @@ partial class SR
 	public const string MDF_TooManyRestrictions = "More restrictions were provided than the requested schema ('{0}') supports.";
 	public const string MDF_DataTableDoesNotExist = "The collection '{0}' is missing from the metadata XML.";
 	public const string MDF_UndefinedCollection = "The requested collection ({0}) is not defined.";
-	public const string MDF_UnsupportedVersion = "The requested collection ({0}) is not supported by this version of the provider.";
+	public const string MDF_UnsupportedVersion = " requested collection ({0}) is not supported by this version of the provider.";
```

Comment:
This should be reverted (the resx seems to be fixed upstream in master).

Reply:
I just checked and the tool has not been touched since it broke; I factored out the revert into a separate PR.